vnext/Microsoft.ReactNative.Cxx/DesktopWindowBridge.h

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <ReactNotificationService.h>
+#include <ReactPropertyBag.h>
+#include <winrt/Windows.Foundation.h>
+#include <functional>
+
+#if defined(USE_WINUI3) && !defined(CORE_ABI) && !defined(__APPLE__) && \
+    WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP | WINAPI_PARTITION_SYSTEM | WINAPI_PARTITION_GAMES)
+
+#include <winrt/Microsoft.ReactNative.h>
+
+namespace Microsoft::ReactNative {
+
+inline auto GetPropertyNameForWindowMessage(UINT uMsg) {
+  auto id = std::to_wstring(uMsg);
+  winrt::Microsoft::ReactNative::ReactPropertyName windowMsgProperty{L"ReactNative.Desktop.WindowMessage", id.c_str()};
+  return windowMsgProperty;
+}
+
+inline void ForwardWindowMessage(
+    const winrt::Microsoft::ReactNative::IReactNotificationService &svc,
+    HWND hwnd,
+    UINT uMsg,
+    WPARAM wParam,
+    LPARAM lParam) {
+  winrt::Microsoft::ReactNative::DesktopWindowMessage args{uMsg, wParam, lParam};
+  svc.SendNotification(GetPropertyNameForWindowMessage(uMsg).Handle(), winrt::box_value((uint64_t)hwnd), args);
+}
+
+inline winrt::Microsoft::ReactNative::IReactNotificationSubscription SubscribeToWindowMessage(
+    const winrt::Microsoft::ReactNative::IReactNotificationService &svc,
+    UINT uMsg,
+    std::function<void(HWND, const winrt::Microsoft::ReactNative::DesktopWindowMessage &)> callback) {
+  return svc.Subscribe(
+      GetPropertyNameForWindowMessage(uMsg).Handle(),
+      nullptr,
+      [=](const winrt::Windows::Foundation::IInspectable &sender,
```

Comment:
>[=] [](start = 6, length = 3)

It is better to avoid the implicit cast like this. It usually causes issues in code outside of unit tests. E.g. here we should use the `callback = std::move(callback)` to efficiently capture the callback.